pacoloco.go

```diff
@@ -125,6 +126,11 @@ func main() {
 		http.DefaultTransport = &http.Transport{Proxy: http.ProxyURL(proxyUrl)}
 	}
 
+	userAgent = "Pacoloco/1.2"
```

Comment:
it hardcodes the pacoloco version. for now, it is ok, but at some point in the future, it needs to be replaced with a build-time set variable.